.gitignore

```diff
@@ -2,6 +2,8 @@
 src/snapred/_version.py
 conda.recipe/
 src/snapred/resources/dev.yml
+docs/source/developer/dao.rst
+docs/source/developer/snapred/*
```

Comment:
During any local build of the docs, approximately 110 files will be created and pollute the git status.  These ignores prevent that from happening